backends-common/opensearch/src/test/java/org/apache/james/backends/opensearch/DockerElasticSearchExtension.java

```diff
@@ -60,4 +61,9 @@ public void awaitForElasticSearch() {
     public DockerElasticSearch getDockerElasticSearch() {
         return elasticSearch;
     }
+
+    @Override
+    public void afterAll(ExtensionContext context) {
+        elasticSearch.stop();
```

Comment:
I'm pretty sure this is worse... did you test it locally? The docker container is usually reused in the module by all test classes... If you stop it after all tests in a class, it's a disaster for the ones after no? But I could be wrong...